hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -230,13 +230,8 @@ public Long getLastResolvedTime() {
 
     @Override
     public String toString() {
-        return "Alert{" +
-                "severity=" + severity +
-                ", status=" + status +
-                ", notes=" + notes +
-                ", lifecycle=" + lifecycle +
-                ", resolvedEvalSets=" + resolvedEvalSets +
-                '}';
+        return "Alert [triggerId=" + getTriggerId() + ", severity=" + severity + ", status=" + status + ", tenantId="
```

Comment:
Hmm, actually, we don't expose any loggers in the API classes, so perhaps that's not a good approach.  Let me  know if you want me to add those fields back...